README.md

```diff
@@ -53,6 +53,26 @@ uv pip install ai-vocabulary-builder
 # uvx --from ai-vocabulary-builder aivoc notebook
 ```
 
+[docker](https://hub.docker.com/_/python)
```

Comment:
It would be nice if you put this section in a separate `CONTRIBUTING.md` file, because only developers will need to build the image themselves. For users, I will upload the image to the docker hub and the README will only mention it via `docker run ...`.